pkg/cloudprovider/fake/controller-kwok/charts/crds/karpenter.sh_nodeclaims.yaml

```diff
@@ -0,0 +1 @@
+../../../pkg/apis/crds/karpenter.sh_nodeclaims.yaml
```

Comment:
This path doesn't look correct since we are now a few directories deeper with this chart

Reply:
Good catch.